config/Mekanism/tiers.toml

```diff
@@ -62,28 +62,28 @@
 	[tier.chemical_tanks]
 		#Storage size of Basic chemical tanks in mB.
 		#Range: 1 ~ 9223372036854775807
-		basicStorage = 64000
+		basicStorage = 5000
```

Comment:
Why are chemical tanks being adjusted? We redid their values in 10.1.0 and I am relatively happy with their updated values.